.cirrus.yml

```diff
@@ -62,7 +66,7 @@ qa_task:
     dockerfile: its/docker/Dockerfile
     docker_arguments:
       CIRRUS_AWS_ACCOUNT: ${CIRRUS_AWS_ACCOUNT}
-      ANT_VERSION: 1.10.12
+      ANT_VERSION: 1.10.13
```

Comment:
isn't this arg just overwriting the one in the image? since we control the image, wouldn't it be better to just define it in the image only?